src/components/Tokens/constants.ts

```diff
@@ -6,6 +6,7 @@ export const LARGE_MEDIA_BREAKPOINT = '840px'
 export const MEDIUM_MEDIA_BREAKPOINT = '720px'
 export const SMALL_MEDIA_BREAKPOINT = '540px'
 export const MOBILE_MEDIA_BREAKPOINT = '420px'
+export const XSMALL_MEDIA_BREAKPOINT = '360px'
```

Comment:
confirming breakpoint size @infredible

Reply:
`396px` is the value for `BREAKPOINTS.xs` in `src/theme/index.tsx`

Honestly these constants should be deprecated in favor off using those later added `BREAKPOINTS` but that is out of the scope of this PR. @LunrEclipse could you add a tech debt ticket to align the Token Explore & Details page with our common theming?